java-frontend/src/main/java/org/sonar/java/JavaSquid.java

```diff
@@ -115,15 +114,15 @@ public void scan(Iterable<File> sourceFiles, Iterable<File> testFiles) {
   }
 
   private void scanSources(Iterable<File> sourceFiles) {
-    Profiler profiler = Profiler.create(LOG).startInfo("Java Main Files AST scan");
+//    Profiler profiler = Profiler.create(LOG).startInfo("Java Main Files AST scan");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)